### ISSUE TITLE
Decide if keep addExistingBridgeToken

### ISSUE THREAD

**Issue by juniuszhou** [Sifchain/sifnode#2793]
The method is used in peggy1.0, we may remove it if not needed anymore. Add unit test for it if keeping it.

**Comment by juniuszhou** [comment]
please @Brando753 @smartyalgo give your suggestion if we should remove the method.

**Comment by Brando753** [comment]
In theory, this is no longer needed; however, if we do deploys on the testnets that accurately mimic production, it will still be needed for Rowan. I say we leave it in.

**Comment by juniuszhou** [comment]
Ok, I will add a unit test for it.

**Comment by juniuszhou** [comment]
close it after #2805 merged.
